.travis.yml

```
@@ -56,6 +56,7 @@ jobs:
             # whether setup.cfg version has "dev" or not, it is added but not duplicated
             ver=$(git show -s --format=%cd --date=format:'%Y%m%d%H%M%S')
             sed -i -E "s/^(version = )([0-9]+\.[0-9]+\.[0-9]+).*$/\1\2.dev$ver/" setup.cfg
+            sed -i -E "s/^(__version__ = )(\"[0-9]+\.[0-9]+\.[0-9]+\").*$/\1\"$RELEASE_VERSION.dev$ver\"/" gravitybee/__init__.py
```

I think if we adjust the second capture group to exclude the quotes, we can use the capture group, making it more like the setup.cfg edit...

```suggestion
            sed -i -E "s/^(__version__ = )\"([0-9]+\.[0-9]+\.[0-9]+)\".*$/\1\"\2.dev$ver\"/" gravitybee/__init__.py
```

---

Okay.  Updated with your suggestion.